product_supplierinfo_group/models/product_supplierinfo.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Akretion France (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from copy import deepcopy
+
+# format: field_from_supplierinfo:field_from_group
+
+MAPPING_RELATED = {
+    "product_tmpl_id": "product_tmpl_id",
+    "name": "partner_id",
+    "product_id": "product_id",
+    "product_name": "product_name",
+    "product_code": "product_code",
+    "sequence": "sequence",
+}
+
+MAPPING_MATCH_GROUP = {
+    "product_tmpl_id": "product_tmpl_id",
+    "name": "partner_id",
+    "product_id": "product_id",
+    "product_name": "product_name",
+    "product_code": "product_code",
+}
+
+
+class ProductSupplierinfo(models.Model):
+    _inherit = "product.supplierinfo"
+
+    supplierinfo_group_id = fields.Many2one("product.supplierinfo.group", required=True)
```

Comment:
I think you maybe needs to set a default (see runbot warning)

Reply:
@rousseldenis thanks for the remark
This meant a bit more head scratching than expected :)
see: https://github.com/OCA/product-attribute/pull/665/commits/46a04187fbd6595c6db4626eb2ef0a4ad0561f94